### PR TITLE
Better client dx

### PR DESCRIPTION
**Pull request by Xavrax**

### CODE REVIEW

File: src/dx/access/builders/mod.rs [hunk 1]
```diff
@@ -19,15 +19,15 @@ pub use revoke::RevokeTokenRequestWithDeserializerBuilder;
 pub use revoke::{RevokeTokenRequest, RevokeTokenRequestBuilder};
 pub mod revoke;
 
+use crate::dx::pubnub_client::PubNubClientInstance;
```
Comment (parfeon) [inline src/dx/access/builders/mod.rs:22]: Isn't `pubnub_client` is redundant in path? I can see that `PubNubClientInstance` is re-exported in `lib.rs`.

**Review by parfeon — APPROVED** [review]
LGTM!

File: src/dx/access/builders/mod.rs (hunk 1 above)
Comment (Xavrax) [reply src/dx/access/builders/mod.rs:22]: I'm gonna check that tomorrow's morning. ;o

Comment (Xavrax) [reply src/dx/access/builders/mod.rs:22]: It's not re-exported. ;o

Comment (parfeon) [reply src/dx/access/builders/mod.rs:22]: Oh, indeed. Should we? :)

Comment (Xavrax) [reply src/dx/access/builders/mod.rs:22]: Imo not. Developers should use it without PubNubMiddleware so we shouldn't reexport that. ;d

**Pull request merged**